ExposurePlayback/Services/AdService/AdServiceEventProvider.swift

```diff
@@ -60,30 +65,92 @@ internal class AdServiceEventProvider: AnalyticsProvider, TimedMetadataProvider,
         
     }
     
     func onTimedMetadataChanged<Tech, Source>(source: Source?, tech: Tech, metadata: [AVMetadataItem]?) where Tech : PlaybackTech, Source : MediaSource {
+        // print(" AdServiceEventProvider onTimedMetadataChanged ")
+        
+        /* if let timeMedata = metadata {
+            for item in timeMedata {
+                        switch item.commonKey {
+
+                        case .commonKeyAlbumName?:
+                            print("AlbumName: \(item.value!)")
+                        case .commonKeyArtist?:
+                            print("Artist: \(item.value!)")
+                        case .commonKeyArtwork?:
+                            print("Artwork: \(item.value!)")
+                        case .commonKeyAuthor?:
+                            print("Author: \(item.value!)")
+                        case .commonKeyContributor?:
+                            print("Contributor: \(item.value!)")
+                        case .commonKeyCopyrights?:
+                            print("Copyrights: \(item.value!)")
+                        case .commonKeyCreationDate?:
+                            print("CreationDate: \(item.value!)")
+                        case .commonKeyCreator?:
+                            print("creator: \(item.value!)")
+                        case .commonKeyDescription?:
+                            print("Description: \(item.value!)")
+                        case .commonKeyFormat?:
+                            print("Format: \(item.value!)")
+                        case .commonKeyIdentifier?:
+                            print("Identifier: \(item.value!)")
+                        case .commonKeyLanguage?:
+                            print("Language: \(item.value!)")
+                        case .commonKeyMake?:
+                            print("Make: \(item.value!)")
+                        case .commonKeyModel?:
+                            print("Model: \(item.value!)")
+                        case .commonKeyPublisher?:
+                            print("Publisher: \(item.value!)")
+                        case .commonKeyRelation?:
+                            print("Relation: \(item.value!)")
+                        case .commonKeySoftware?:
+                            print("Software: \(item.value!)")
+                        case .commonKeySubject?:
+                            print("Subject: \(item.value!)")
+                        case .commonKeyTitle?:
+                            print("Title: \(item.value!)")
+                        case .commonKeyType?:
+                            print("Type: \(item.value!)")
+
+                        case .id3MetadataKeyAlbumTitle?:
+                            print("id3MetadataKeyAlbumTitle: \(item.value!)")
+
+                        default:
+                            print("other data: \(item.value!)")
+                        }
+                    }
+        } */
+
+        
+        
+        
```

Comment:
Do we need all those comments here and above ?

Reply:
Probably not , will remove those . Good catch